crypto/src/main/java/org/springframework/security/crypto/password/DelegatingPasswordEncoder.java

```diff
@@ -286,9 +286,17 @@ public String encode(CharSequence rawPassword) {
 		@Override
 		public boolean matches(CharSequence rawPassword, String prefixEncodedPassword) {
```

Comment:
Will you please update the copyright year on the files you edit to end in `2024`?

Reply:
I have made changes to the files to ensure that the copyright year is up to date.